metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -252,13 +257,24 @@ def _extract_record(
             topic, topic_detail, extra_topic_config
         )
 
+        # 4. Set dataset's description as top level doc, if topic schema type is avro
+        description = None
+        if (
+            schema_metadata is not None
+            and isinstance(schema_metadata.platformSchema, KafkaSchemaClass)
+            and schema_metadata.platformSchema.documentSchemaType == "AVRO"
```

Comment:
at debug level log platformSchema; ignore if already get log

Reply:
I don't see any debug level log on platformSchema in any other sources